test/promql/tests/data_plane/gitops_sync_test.yaml

```diff
@@ -0,0 +1,666 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
+  # -----------------------------------------------------------------------------------
+  # The following section describes tests for the situation where no apps are Synced
+  # -----------------------------------------------------------------------------------
+  - interval: 1m
+    input_series:
+
+      # Compare this test with the one for GitOpsApplicationNoSyncApps above
```

Comment:
This comment is referencing a non-existing alert

Reply:
Good catch. Comment removed.